crates/bevy_diagnostic/src/diagnostic.rs

```diff
@@ -115,6 +115,22 @@ impl Diagnostic {
         }
     }
 
+    /// Return the minimum of this diagnostic's values.
+    pub fn min(&self) -> Option<f64> {
```

Comment:
We could calculate min/max in `add_measurement` like we already do for the sum.

Reply:
Ya, this was a bit of a first pass on this, it'd probably be better to do the same as with the average here. Will work on this in a bit.